ops/pebble.py

```diff
@@ -1966,3 +1952,162 @@ def send_signal(self, sig: typing.Union[int, str], services: typing.List[str]):
             'services': services,
         }
         self._request('POST', '/v1/signals', body=body)
+
+
+class MultipartLargeFileParser:
```

Comment:
Nit: this is used for all files, right, not just large ones? What about removing `Large` from the name?

Reply:
Renamed.